src/main/java/games/strategy/engine/config/GameEnginePropertyReader.java

```diff
@@ -0,0 +1,71 @@
+package games.strategy.engine.config;
+
+import java.io.File;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+import com.google.common.annotations.VisibleForTesting;
+
+import games.strategy.util.Version;
+
+/**
+ * Reads property values from the game engine configuration file.
+ */
+public class GameEnginePropertyReader extends PropertyFileReader {
+
+  public static final String GAME_ENGINE_PROPERTY_FILE = "game_engine.properties";
+
+  public GameEnginePropertyReader() {
+    super(GAME_ENGINE_PROPERTY_FILE);
+  }
+
+  @VisibleForTesting GameEnginePropertyReader(File propFile) {
+    super(propFile);
+  }
+
+  public Version readEngineVersion() {
+    return new Version(readProperty(GameEngineProperty.ENGINE_VERSION));
+  }
+
+  public String readLobbyPropertiesUrl() {
+    final String url = readProperty(GameEngineProperty.LOBBY_PROPS_URL);
+    try {
+      return getUrlFollowingRedirects(url);
+    } catch (Exception e) {
+      throw new RuntimeException("Failed to get download file url: " + url, e);
+    }
+  }
+
+  private static String getUrlFollowingRedirects(final String possibleRedirectionUrl) throws Exception {
```

Comment:
Not the scope of tis PR, but this Method is not going to work as expected if the URl being redirected to is going to redirect the User again.
Using a do while loop could fix this issue, but it does not handle circular redirects, e.g redirects that will never end.
We should use the Apache HttpClient here